src/components/HelmRepositories/NewRepositoryDialog.tsx

```diff
@@ -83,14 +72,15 @@ const NewRepositoryDialog = ({ closeDialog, onCreate }: NewRepositoryDialogProps
 
     try {
       const secret = await createArgoCDSecret;
+      if (formValues.type === 'helm') {
+        await refetch();
+      }
+      await refetch();
```

Comment:
so in case of `helm`, the refetch happens twice..
Seems like the `if` part is not needed anymore ?